packages/amplify-cli-core/package.json

```diff
@@ -7,6 +7,9 @@
     "url": "https://github.com/aws-amplify/amplify-cli.git",
     "directory": "packages/amplify-cli-core"
   },
+  "publishConfig": {
```

Comment:
why is this added ?. I think we only need this for scoped packages

Reply:
Staging ground for future work. see https://github.com/aws-amplify/amplify-cli/pull/12235#discussion_r1137502614 .